controllers/chat/deleteMessage.js

```diff
@@ -0,0 +1,32 @@
+const {StreamChat} = require('stream-chat');
+
+const deleteMessage = async (req, res) => {
+  const client = new StreamChat(process.env.API_KEY, process.env.SECRET_KEY);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoding API keys directly in the code is not a secure practice. It's better to use a configuration file or environment variables to store sensitive information like API keys.
```diff
- const client = new StreamChat(process.env.API_KEY, process.env.SECRET_KEY);
+ const client = new StreamChat(process.env.STREAM_API_KEY, process.env.STREAM_SECRET_KEY);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->